### PR TITLE
Suggesting improvements

### PR DESCRIPTION
**Pull request by mariajdab**
Some improvements involves: 

- `offchaindatahandler`: keeping all receiver names equal
- `p2p`: keeping all receiver names equal
- `cli`: checking errors
- `voconed`: checking errors
`start`: use error variable already declared
- fix comments

### CODE REVIEW

File: cmd/cli/vocdonicli.go [hunk 1]
```diff
@@ -114,7 +114,11 @@ func (v *vocdoniCLI) setHost(host string) error {
 	if err != nil {
 		return err
 	}
-	v.api.SetHostAddr(u)
+	err = v.api.SetHostAddr(u)
```
Comment (p4u) [inline cmd/cli/vocdonicli.go]: Hey Maria.

It is generally recommended to use the short variable declaration operator := when initializing a new variable in the same line that it is used. This creates the variable in the closest scope possible, which can help make the code more readable and reduce the chance of bugs. Using := also helps to avoid reusing variables that might have been defined earlier in the code and might have different values.

Additionally, it is considered a good practice to avoid reusing error variables, as it can lead to confusion and make it harder to understand the code. Instead, it is recommended to create a new variable for each error that is returned, and to name the variable something that clearly indicates its purpose.

To sum it up, it is suggested to use `if err := v.api.SetHostAddr(u)` instead of `err = v.api.SetHostAddr(u)` to keep the code more readable, and avoid potential bugs and confusion.

Please apply this principle to the following modified lines.

File: vochain/offchaindatahandler/census.go [hunk 2]
```diff
@@ -9,29 +9,29 @@ import (
 )
 
 // importExternalCensus imports a census from a remote URI into the censusDB storage.
-func (d *OffChainDataHandler) importExternalCensus(uri string, data []byte) {
+func (c *OffChainDataHandler) importExternalCensus(uri string, data []byte) {
```
Comment (p4u) [inline vochain/offchaindatahandler/census.go]: What is your reasoning for using `c` instead of `d`? If it refers to chain, I must say the important concept for this name is Data.

**Review by p4u — CHANGES_REQUESTED** [review]

**Comment by p4u** [comment]
Regarding the failing compose test, its nothing related to your changes.

File: vochain/offchaindatahandler/census.go (hunk 2 above)
Comment (mariajdab) [reply vochain/offchaindatahandler/census.go]: My first approach was checking the most common receiver name for `OffChainDataHandler`, and you are using thirteen times `c` vs three times `d` and one time `s`

File: cmd/cli/vocdonicli.go (hunk 1 above)
Comment (mariajdab) [reply cmd/cli/vocdonicli.go]: Hey Pau! 

I get it, thank you! I will make the changes.

File: vochain/offchaindatahandler/census.go (hunk 2 above)
Comment (mariajdab) [reply vochain/offchaindatahandler/census.go]: I will update it to use `d` :raised_hands:

Comment (mariajdab) [reply vochain/offchaindatahandler/census.go:12]: I will update it to use `d` :raised_hands:

**Comment by p4u** [comment]
Please rebase your branch, the CI issue should be fixed

**Review by p4u — APPROVED** [review]
